charts/ccd-data-store-api/values.yaml

```diff
@@ -60,3 +60,6 @@ java:
     HTTP_CLIENT_SECONDS_IDLE_CONNECTION: '120'
     HTTP_CLIENT_MAX_CLIENT_PER_ROUTE: '20'
     HTTP_CLIENT_VALIDATE_AFTER_INACTIVITY: '2000'
+
+    DATA_STORE_ROLE_LOGGING_ENABLED: 'false'
```

Comment:
I think instead of switching the logging on and off, we should just change the logging level per package/class.